### ISSUE TITLE
Admin Integration Expects Global jQuery?

### ISSUE THREAD

**Issue by jpulec** [furious-luke/django-address#46]
I've been trying to get the autocomplete to work in the django admin, but cannot. I've correctly set up my `ModelAdmin` as so:

``` python
from address.forms import AddressWidget
from address.models import AddressField

class SavedDeliveryLocationAdmin(admin.ModelAdmin):
  formfield_overrides = {
    AddressField: {'widget': AddressWidget()}
  }
```

but when I open the admin, my console tells me that `$ is not a function` for `address.js` and `Cannot read property 'extend' of undefined` for `jquery.geocomplete.js`. It would appear that these files expect `jQuery` to be a global, but it is not.

I'm using django 1.9.5.

**Comment by ixeft** [comment]
Same here

**Comment by banagale** [comment]
Assigning this to 0.2.3. If this is still broken or improperly implemented in `master` we'll fix it before the big changes to 0.3.0/

**Comment by banagale** [comment]
Looked into this and tested it. This seems to have been fixed in @karbassi's [2018 commit](https://github.com/furious-luke/django-address/commit/06909259dea7641aed77b2ba32f82d68c70ac06a#diff-e4017b7c40b10f1019415da5e22a2c9b).
